images/win/Windows2019-Readme.md

```diff
@@ -1,107 +1,122 @@
+| [[all OSs] Java 7 will be removed from images on February, 8.](https://github.com/actions/virtual-environments/issues/2446) |
+| [[all OS versions] Android NDK is updated to r22.](https://github.com/actions/virtual-environments/issues/2420) |
+| [[Windows] Default PHP is set to 8.0](https://github.com/actions/virtual-environments/issues/2416) |
+***
 # Microsoft Windows Server 2019 Datacenter
-- OS Version: 10.0.17763 Build 1637
-- Image Version: 20210110.1
+- OS Version: 10.0.17763 Build 1697
+- Image Version: 20210118.1
 
 ## Enabled windows optional features
 - Windows Subsystem for Linux [WSLv1]
 
 ## Installed Software
 ### Language and Runtime
-- Java 1.7.0_232 
-- Java 1.8.0_275 (default)
-- Java 11.0.9.1 
-- Java 13.0.2 
 - Bash 4.4.23(1)-release
-- Python 3.7.9
-- Ruby 2.5.8p224
 - Go 1.14.13
-- PHP 8.0.1
+- Java 1.7.0_232  Java 1.8.0_275 (default) Java 11.0.9.1  Java 13.0.2 
 - Julia 1.5.3
-- Perl 5.32.0
 - Node 14.15.4
+- Perl 5.32.0
+- PHP 8.0.1
+- Python 3.7.9
+- Ruby 2.5.8p224
 
 ### Package Management
 - Chocolatey 0.10.15
-- Vcpkg 2020.11.12 (build from master \<a14a6bc>)
+- Composer 2.0.8
+- Helm 3.5.0
+- Miniconda 4.9.2
 - NPM 6.14.10
-- Yarn 1.22.10
+- NuGet 5.8.1.7021
 - pip 20.3.3 (python 3.7)
-- Miniconda 4.9.2
+- Pipx 0.16.0.0
 - RubyGems 2.7.6.2
-- Helm 3.4.2
-- Composer 2.0.8
-- NuGet 5.8.0.6930
-- Pipx 0.15.6.0
+- Vcpkg 2020.11.12 (build from master \<0c94afc>)
+- Yarn 1.22.10
 
 ### Project Management
 - Ant 1.10.9
+- Gradle 6.8
 - Maven 3.6.3
-- Gradle 6.7
 - sbt 1.4.5
 
 ### Tools
+- 7zip 19.00
 - azcopy 10.8.0
 - Bazel 3.7.2
 - Bazelisk 1.7.3
-- CMake 3.19.2
+- Cabal 3.2.0.0
+- CMake 3.19.3
 - CodeQL Action Bundle 2.4.0
-- R 4.0.3
 - Docker 19.03.14
 - Docker-compose 1.27.4
+- ghc 8.10.3
 - Git 2.30.0
-- Git LFS 2.13.1
-- Google Cloud SDK 322.0.0
+- Git LFS 2.13.2
+- Google Cloud SDK 323.0.0
 - InnoSetup 6.1.2
 - jq 1.6
-- Kubectl 1.20.0
 - Kind 0.9.0
-- Mingw-w64 8.1.0
+- Kubectl 1.20.2
 - Mercurial 5.0
-- NSIS v3.06.1
+- Mingw-w64 8.1.0
 - Newman 5.2.2
+- NSIS v3.06.1
 - OpenSSL 1.1.1
 - Packer 1.6.6
-- Pulumi v2.17.0
-- Subversion (SVN) 1.14.0
-- ghc 8.10.3
-- Cabal 3.2.0.0
+- Pulumi v2.17.2
+- R 4.0.3
 - Stack 2.5.1
-- WinAppDriver 1.2.2009.02003
-- zstd 1.4.5
+- Subversion (SVN) 1.14.0
 - VSWhere 2.8.4
-- 7zip 19.00
+- WinAppDriver 1.2.2009.02003
 - yamllint 1.25.0
+- zstd 1.4.5
 
 ### CLI Tools
-- Azure CLI 2.17.1
-- Azure DevOps CLI extension 0.18.0
-- Azure Dev Spaces CLI 1.0.20201219.2
-- AWS CLI 2.1.17
+- Alibaba Cloud CLI 3.0.66
+- AWS CLI 2.1.19
 - AWS SAM CLI 1.15.0
 - AWS Session Manager CLI 1.2.30.0
-- Alibaba Cloud CLI 3.0.65
+- Azure CLI 2.17.1
+- Azure Dev Spaces CLI 1.0.20201219.2
+- Azure DevOps CLI extension 0.18.0
 - Cloud Foundry CLI 6.53.0
-- Hub CLI 2.14.2
 - GitHub CLI 1.4.0
+- Hub CLI 2.14.2
 
 ### Rust Tools
+- Cargo 1.49.0
 - Rust 1.49.0
+- Rustdoc 1.49.0
+- Rustup 1.23.1
 
 #### Packages
 - bindgen 0.56.0
-- cbindgen 0.16.0
```

Comment:
looks like we lost it after these changes [[Windows] Sort tool names in README #2422](https://github.com/actions/virtual-environments/pull/2422/files#diff-0f0deef55edf9ac41fe9db1cd386e44aac4536569e49e147e8fdc52457ca8dedL127)

Reply:
Yes, @dibir-magomedsaygitov could you please prepare short fix?

Reply:
Sorry, prepared fix. See [#2482](https://github.com/actions/virtual-environments/pull/2482)